### PR TITLE
Test

### PR DESCRIPTION
**Pull request by janisz**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,92 @@
+apply plugin: 'java'
+apply plugin: 'groovy'
+apply plugin: 'antlr'
+apply plugin: 'idea'
+apply plugin: 'jacoco'
+apply plugin: 'com.github.kt3k.coveralls'
```
Comment (pjagielski) [inline build.gradle]: test

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 2]
```diff
@@ -0,0 +1,395 @@
+package pl.edu.pw.mini.msi.knowledgerepresentation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.ActionDomain;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.sentenceParts.*;
```
Comment (pjagielski) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:6]: test

Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:6]: [Checkstyle] WARNING: Using the '.*' form of import should be avoided - pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.sentenceParts.*.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 3]
```diff
@@ -0,0 +1,395 @@
+package pl.edu.pw.mini.msi.knowledgerepresentation;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.ActionDomain;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.sentenceParts.*;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.sentences.*;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:7]: [Checkstyle] WARNING: Using the '.*' form of import should be avoided - pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.sentences.*.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 4]
```diff
@@ -0,0 +1,395 @@
...
+import org.slf4j.LoggerFactory;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.ActionDomain;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.sentenceParts.*;
+import pl.edu.pw.mini.msi.knowledgerepresentation.actionDomain.sentences.*;
+import pl.edu.pw.mini.msi.knowledgerepresentation.grammar.ActionLanguageBaseListener;
+import pl.edu.pw.mini.msi.knowledgerepresentation.grammar.ActionLanguageParser;
+
+public class ActionLanguageListener extends ActionLanguageBaseListener {
+
+    private static final Logger log = LoggerFactory.getLogger(ActionLanguageListener.class);
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:13]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 5]
```diff
@@ -0,0 +1,395 @@
...
+
+    private Action lastAction = null;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:15]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 6]
```diff
@@ -0,0 +1,395 @@
...
+    private Action preLastAction = null;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:16]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 7]
```diff
@@ -0,0 +1,395 @@
...
+    private Actor lastActor = null;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:17]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 8]
```diff
@@ -0,0 +1,395 @@
...
+    //private ActionLanguageParser.LogicalExpressionContext lastLogExpr = null;
+    private EQueryType lastQueryType = EQueryType.always;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:19]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 9]
```diff
@@ -0,0 +1,395 @@
...
+    private Fluent lastFluent = null;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:20]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 10]
```diff
@@ -0,0 +1,395 @@
...
+    private IFormula lastFormula = null;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:21]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 11]
```diff
@@ -0,0 +1,395 @@
...
+    private ScenarioName lastScenarioName = null;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:22]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 12]
```diff
@@ -0,0 +1,395 @@
...
+    private Time lastTime = null;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:23]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 13]
```diff
@@ -0,0 +1,395 @@
...
+    private boolean lastTypically = false;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:24]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 14]
```diff
@@ -0,0 +1,395 @@
...
+
+
+    private boolean isInLogicalExpression = false;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:27]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 15]
```diff
@@ -0,0 +1,395 @@
...
+    private IFormula lastUnderConditionFormula = null;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:28]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 16]
```diff
@@ -0,0 +1,395 @@
...
+    private boolean isInUnderCondition = false;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:29]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 17]
```diff
@@ -0,0 +1,395 @@
...
+
+    //===========================================================================================
+    private ActionDomain actionDomain = new ActionDomain();
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:32]: [Checkstyle] WARNING: 'member def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 18]
```diff
@@ -0,0 +1,395 @@
...
+
+
+    public ActionLanguageListener(ActionDomain actionDomain) {
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:35]: [Checkstyle] WARNING: 'ctor def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 19]
```diff
@@ -0,0 +1,395 @@
...
+        this.actionDomain = actionDomain;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:36]: [Checkstyle] WARNING: 'ctor def' child have incorrect indentation level 8, expected level should be 4.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 20]
```diff
@@ -0,0 +1,395 @@
...
+        log.debug("ActionLanguageListener(ActionDomain actionDomain)");
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:37]: [Checkstyle] WARNING: 'ctor def' child have incorrect indentation level 8, expected level should be 4.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 21]
```diff
@@ -0,0 +1,395 @@
...
+    }
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:38]: [Checkstyle] WARNING: 'ctor def rcurly' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 22]
```diff
@@ -0,0 +1,395 @@
...
+
+    public ActionDomain getActionDomain() {
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:40]: [Checkstyle] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2.

Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:40]: [Checkstyle] WARNING: Missing a Javadoc comment.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 23]
```diff
@@ -0,0 +1,395 @@
...
+        //actionDomain.baseSentences.add(...);
+        //actionDomain.scenarios.add(...);
+        //actionDomain.addQueries(...);
+        actionDomain.calculateFullScenarios();
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:44]: [Checkstyle] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 24]
```diff
@@ -0,0 +1,395 @@
...
+        actionDomain.calculateMappedQueries();
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:45]: [Checkstyle] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 25]
```diff
@@ -0,0 +1,395 @@
...
+        actionDomain.fillFluentsAndActionsIDs();
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:46]: [Checkstyle] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 26]
```diff
@@ -0,0 +1,395 @@
...
+
+        return actionDomain;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:48]: [Checkstyle] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 27]
```diff
@@ -0,0 +1,395 @@
...
+    }
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:49]: [Checkstyle] WARNING: 'method def rcurly' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 28]
```diff
@@ -0,0 +1,395 @@
...
+    //ActionLanguageParser.LogicalExpressionContext ctx;
+    //===========================================================================================
+
+    /**
+     * {@inheritDoc}
+     * <p/>
+     * <p>The default implementation does nothing.</p>
+     */
+    @Override
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:58]: [Checkstyle] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 29]
```diff
@@ -0,0 +1,395 @@
...
+    public void enterProgramm(ActionLanguageParser.ProgrammContext ctx) {
+    }
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:60]: [Checkstyle] WARNING: 'method def rcurly' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 30]
```diff
@@ -0,0 +1,395 @@
...
+
+    /**
+     * {@inheritDoc}
+     * <p/>
+     * <p>The default implementation does nothing.</p>
+     */
+    @Override
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:67]: [Checkstyle] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 31]
```diff
@@ -0,0 +1,395 @@
...
+    public void exitProgramm(ActionLanguageParser.ProgrammContext ctx) {
+    }
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:69]: [Checkstyle] WARNING: 'method def rcurly' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 32]
```diff
@@ -0,0 +1,395 @@
...
+
+    private void clear(boolean clearLastScenarioName) {
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:71]: [Checkstyle] WARNING: 'method def modifier' have incorrect indentation level 4, expected level should be 2.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 33]
```diff
@@ -0,0 +1,395 @@
...
+        lastAction = null;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:72]: [Checkstyle] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 34]
```diff
@@ -0,0 +1,395 @@
...
+        preLastAction = null;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:73]: [Checkstyle] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 35]
```diff
@@ -0,0 +1,395 @@
...
+        lastActor = null;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:74]: [Checkstyle] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4.

File: src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java [hunk 36]
```diff
@@ -0,0 +1,395 @@
...
+        //ActionLanguageParser.LogicalExpressionContext lastLogExpr = null;
+        lastQueryType = null;
```
Comment (sputnikci) [inline src/main/java/pl/edu/pw/mini/msi/knowledgerepresentation/ActionLanguageListener.java:76]: [Checkstyle] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4.

**Pull request closed without merging**